docs/content/maintenance/system-tables.md

```diff
@@ -406,4 +406,23 @@ SELECT * FROM T$statistics;
 1 rows in set
 */
 ```
+### Binlog Table
+
+You can streaming or batch query the binlog through binlog table. In this system table,
+the update before and update after will be packed in one row. 
+
+```
+/*
++------------------+----------------------+-----------------------+
+|     rowkind      |     column_0         |     column_1          |
++------------------+----------------------+-----------------------+
+|        +I        |  [col_0, null]       |   [col_1, null]       |
```

Comment:
Why not just one element?

Reply:
The schema should keep same, so this is an array only with one element.

Reply:
I mean one element, not one element and a null, two elements.

Reply:
Get it, updated.